en/advanced/yggdrasil.md

```diff
@@ -1,79 +1,89 @@
 
-你可以在 [https://authlib-injector.yushi.moe/~download/](https://authlib-injector.yushi.moe/~download/) 下载到最新版本的 authlib-injector。
+You can download the latest version of authlib-injector at <https://authlib-injector.yushi.moe/~download/>
 
-## Yggdrasil API 地址
-LittleSkin 的 Yggdrasil API 的地址是：
+## Yggdrasil API root
+
+The Yggdrasil API root of LittleSkin is:
 
 ```
 https://littleskin.cn/api/yggdrasil
 ```
 
-LittleSkin 已在全站启用 authlib-injector 的 API 地址指示（ALI）功能。在使用支持 ALI 的 authlib-injector 和启动器时，输入 LittleSkin 的任意页面的地址即可被识别。
+LittleSkin supports API Location Indication (ALI) feature of authlib-injector, so it's okay to just enter any URL of LittleSkin when using launchers and authlib-injector supports ALI and it will be identified.
 
-## API 文档
+## API Documents
 
-请参阅：[Yggdrasil 服务端技术规范](https://github.com/yushijinhun/authlib-injector/wiki/Yggdrasil%E6%9C%8D%E5%8A%A1%E7%AB%AF%E6%8A%80%E6%9C%AF%E8%A7%84%E8%8C%83)
+See: [Yggdrasil Server Technical Specifications](https://github.com/yushijinhun/authlib-injector/wiki/Yggdrasil%E6%9C%8D%E5%8A%A1%E7%AB%AF%E6%8A%80%E6%9C%AF%E8%A7%84%E8%8C%83)
 
-## 在服务端使用
+## Using in the server
 
-请先将服务器配置文件（一般为 server.properties）中 `online-mode` 一项的值设为 `true`，然后在你的服务端的启动指令的 `-jar` 参数前添加如下参数：
+Please set `online-mode` to `true` in your server configuration file (typically `server.properties`) first. Then add the following parameter to your startup command before the `-jar` parameter:
 
 ```
 -javaagent:{path/to/authlib-injector.jar}=https://littleskin.cn/api/yggdrasil
 ```
 
-其中 `{path/to/authlib-injector.jar}` 为指向 authlib-injector 的路径。
+The `{path/to/authlib-injector.jar}` is the path that directs to the authlib-injector.
+
 
 ::: warning
-<strong>实际填写的路径两边没有大括号！</strong>大括号只是表明必须正确指定这个参数的值，但是实际上并不需要填入大括号。
+<strong>DO NOT include parentheses at either end!<strong> The big parentheses are only used to show that you must assign this value correctly, but needn't add the big paratheses in real.
 :::
 
 ::: tip
-如果你使用 BungeeCord，你需要在所有服务端（包括 BungeeCord 和所有子服）中加载 authlib-injector（方法见上），但只有 BungeeCord 可以打开 `online-mode`，其他服务端应该关闭 `online-mode`。
+For BungeeCord users, make sure you have authlib-injector loaded on every server instance (including BungeeCord and all sub servers). But you should enable `online-mode` for BungeeCord ONLY, keep it disabled for all sub servers.
 :::
 
-## 在客户端使用
+## Using in the client
 
-在客户端中使用 LittleSkin 的 Yggdrasil 需要启动器支持自定义 Yggdrasil 服务器。推荐使用 [HMCL 3](https://www.mcbbs.net/thread-142335-1-1.html) 和 [BakaXL](https://www.mcbbs.net/thread-512144-1-1.html)。
+You need a launcher which supports using a custom Yggdrasil server so that you can use the Yggdrasil service of LittleSkin in the client. [HMCL 3](https://www.mcbbs.net/thread-142335-1-1.html) and [BakaXL](https://www.mcbbs.net/thread-512144-1-1.html) is recommended.
```

Comment:
```suggestion
You need a launcher which supports using a custom Yggdrasil server so that you can use the Yggdrasil service of LittleSkin in the client. [HMCL 3](https://www.mcbbs.net/thread-142335-1-1.html) and [BakaXL](https://www.mcbbs.net/thread-512144-1-1.html) are recommended.
```